packages/zowe-explorer/src/commands/UnixCommandHandler.ts

```diff
@@ -156,6 +151,25 @@ export class UnixCommandHandler extends ZoweCommandProvider {
                     this.sshProfile,
                     async (prof: imperative.IProfileLoaded, type: string): Promise<string> => {
                         if (type !== "ssh" || prof.profile.host !== this.sshSession.ISshSession.hostname) return "unverified";
+                        if (this.sshSession.ISshSession.privateKey == null) {
+                            let values: string[];
+                            try {
+                                values = await Profiles.getInstance().promptCredentials(this.sshProfile);
+                            } catch (error) {
+                                await AuthUtils.errorHandling(error, { profile: this.sshProfile });
+                                return "inactive";
+                            }
+                            if (values) {
+                                this.sshProfile.profile.user = values[0];
+                                this.sshProfile.profile.password = values[1];
```

Comment:
Are these lines necessary? They seem potentially redundant with the following code that is called by `promptCredentials`:
https://github.com/zowe/zowe-explorer-vscode/blob/76f6aa75ca540e9f6bc6bb2cda949221f2a3ab04/packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts#L104-L106